### ISSUE TITLE
EXAMPLE_COUNTS transposition in eval_reward_bench_bt.py silently distorts the reported Safety score

### ISSUE THREAD

**Issue by shaurya416** [RLHFlow/RLHF-Reward-Modeling#63]
`useful_code/eval_reward_bench_bt.py:130-131`

```python
EXAMPLE_COUNTS = {
    ...
    "xstest-should-refuse": 250,
    "xstest-should-respond": 154,
    ...
}
```

These two values are swapped. The correct counts (matching upstream `allenai/reward-bench`'s `rewardbench/constants.py` and this repo's own sibling script `useful_code/eval_reward_bench_pm.py:129-130`) are `xstest-should-refuse: 154, xstest-should-respond: 250`.

`EXAMPLE_COUNTS` is used only as a per-subset weight table inside `calculate_scores_per_section()` (lines 176-189), which builds the composite `Safety` category score reported at the end of every run as a weighted mean of per-subset accuracy. Swapping the two weights misattributes weight between the two XSTest subsets whenever the model's accuracy differs between them — silently, with no exception or warning.

## Measured

Function body copied verbatim into plain Python (the package is classifier-gated and was not pip installed):

| Case | Input | bt.py (transposed) | pm.py weights (correct) | Result |
|---|---|---|---|---|
| Control — sanity | Uniform 0.90 accuracy across all 5 Safety subsets | 90.0 | 90.0 | Equal, as required when weighting cannot matter |
| **Real reproduction** — GitHub issue #13's actual eval run (RM-Gemma-2B) | Measured accuracies: refusals-dangerous=0.070000, refusals-offensive=0.240000, xstest-should-refuse=0.422078, xstest-should-respond=0.940000, donotanswer=0.257353 | **42.74** (matches the number the issue reporter actually saw) | **49.46** | 6.72-point silent understatement |
| Control — sign flip | Same subsets, xstest skew reversed (refuse=0.99, respond=0.60) | 89.11 | 84.05 | Distortion sign flips, confirming genuine misattribution rather than a fixed offset |

The second row reproduces a real, previously-filed report: issue #13 ("Low Safety Score for RM-Gemma-2B Model") shows a run whose printed per-subset table already carries the correct counts (n=154 for xstest-should-refuse, n=250 for xstest-should-respond — i.e., the ground truth, visible right there in the DataFrame the script itself prints), yet the weighted Safety score it reports, 42.74, was computed with the transposed `EXAMPLE_COUNTS`. The issue was closed by attributing the low score entirely to the model's training data; the weight-table bug was never identified.

## Consequence

Every reported `Safety` composite score from `eval_reward_bench_bt.py` — the repo's Bradley-Terry reward-model evaluator — is computed with the wrong per-subset weighting whenever a model's XSTest-refuse and XSTest-respond accuracies diverge (the common case, since the two subsets test opposite behaviors: refusing unsafe prompts vs. answering safe-looking ones). The distortion is silent, direction-dependent, and large enough (multiple points, up to ~7 points measured here) to change model rankings or mask a real safety regression, with no signal to the person reading the printed table that anything is wrong.

## Suggested fix

Apply the same fix already merged into the sibling script: PR #56 changed `eval_reward_bench_pm.py`'s `EXAMPLE_COUNTS` from `xstest-should-refuse: 250, xstest-should-respond: 154` to `xstest-should-refuse: 154, xstest-should-respond: 250`, citing upstream `allenai/reward-bench` commit `6552e932` as the source of truth. `eval_reward_bench_bt.py` was never given the equivalent one-line fix and still carries the pre-PR-#56 values.

Happy to open the PR.
